src/centralserver/admin-ui/src/views/SecurityServers/SecurityServersList.vue

```diff
@@ -41,17 +41,21 @@
       :loading="loading"
       :headers="headers"
       :items="securityServers"
-      :search="search"
       :must-sort="true"
-      :items-per-page="-1"
+      :options.sync="pagingSortingOptions"
```

Comment:
Uses vuetify default paging options 5/10/15/ALL, ALL produces http 500, options should be consolidated with XRDDEV-1910 which uses 10/25

Reply:
Addressed in https://github.com/nordic-institute/X-Road/commit/af99d4d47bca9b426372304ff4e5c69309d28b37